.github/workflows/security-scan.yml

```
@@ -9,42 +9,242 @@ on:
+
+      - name: Evaluate SAST results
+        if: always()
+        run: |
+          # Check JSON results for actual ERROR-severity findings rather than
+          # relying on exit codes (which also fail on parse errors).
+          FAILED=false
+          for f in semgrep-python.json semgrep-typescript.json semgrep-kotlin.json; do
+            if [ ! -f "$f" ]; then continue; fi
+            LANG=$(echo "$f" | sed 's/semgrep-//;s/.json//')
+            COUNT=$(python3 -c "
+import json, sys
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)